external/fv3fit/tests/emulation/test_train_microphysics.py

```diff
@@ -146,37 +128,6 @@ def test_TrainConfig_from_args_sysargv(monkeypatch):
     assert config.model.architecture.name == "rnn"
 
 
-def test_TrainConfig_invalid_input_vars():
```

Comment:
This is impossible to happen now since the variables are now inferred from the model config.